server/src/main/java/org/opensearch/threadpool/ThreadPool.java

```diff
@@ -142,8 +143,8 @@ public static ThreadPoolType fromType(String type) {
         map.put(Names.LISTENER, ThreadPoolType.FIXED);
         map.put(Names.GET, ThreadPoolType.FIXED);
         map.put(Names.ANALYZE, ThreadPoolType.FIXED);
-        map.put(Names.WRITE, ThreadPoolType.FIXED);
-        map.put(Names.SEARCH, ThreadPoolType.FIXED_AUTO_QUEUE_SIZE);
+        map.put(Names.WRITE, ThreadPoolType.RESIZABLE);
```

Comment:
Do we really need to change `WRITE` threadpool as they have been moved to support byte-size based defaulting to 10k

Reply:
that is a very good point. yes, there is actually no need to change write to resizable anymore